plumed/variants.yaml

```
@@ -0,0 +1,11 @@
+mpi:
+  - mpich
+  - openmpi
+  - nompi
+# These are only for [osx and x86_64]
+c_stdlib:
+- macosx_deployment_target
```

```suggestion
  - macosx_deployment_target
```

or am I misunderstanding the syntax?